primefaces/src/main/resources/META-INF/resources/primefaces/datepicker/0-datepicker.js

```diff
@@ -278,6 +278,11 @@
             return firstDayOfWeek > 0 ? 7 - firstDayOfWeek : 0;
         },
 
+        getSaturdayIndex: function () {
+            var firstDayOfWeek = this.options.locale.firstDay !== undefined ? this.options.locale.firstDay : this.options.locale.firstDayOfWeek;
```

Comment:
I know it's a copied function, but isn't this the same?
```suggestion
            var firstDayOfWeek = this.options.locale.firstDay || this.options.locale.firstDayOfWeek;
```